src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1415,6 +1424,32 @@ public function cancellationAction(int $orderId, Request $request)
         ]);
     }
 
+    /**
+     * @AdminSecurity("is_granted('update', request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return JsonResponse
+     */
+    public function configureProductPaginationAction(Request $request): JsonResponse
+    {
+        $numPerPage = (int) $request->request->get('numPerPage');
+        if ($numPerPage < 1) {
+            $numPerPage = 10;
```

Comment:
Idem here (hardcoded value), it would be nice if those values were defined somewhere (as a const in this class, or somewhere else?)